bin/hl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+
+const fs = require('fs-extra');
+const program = require('commander');
+const hl = require('../');
+
+program
+  .command('create')
+  .alias('c')
+  .description('Create a hashlink given a URL and/or data.')
+  .option('-u, --url <location>', 'target URL for the hashlink')
+  .option('-l, --legacy', 'create a hashlink for a legacy URL')
+  .option('-h, --hash <algorithm>', 'cryptographic hash algorithm', 'sha2-256')
+  .option('-b, --base <encoding>', 'base encoding', 'base58-btc')
+  .action(async (dataFile, options) => {
+    try {
+      let data, urls, transforms, meta = undefined;
+      // read data from disk if provided
+      if(dataFile) {
+        data = fs.readFileSync(dataFile, null);
```

Comment:
This is adding an extra null byte to the end of the buffer, which is wrong. Thoughts @dlongley @davidlehn?

Reply:
Nevermind, it was a stray `\n`, fixed in tests.